ubg/src/mafia/MafiaDay.js

```diff
@@ -165,71 +138,58 @@ function MafiaDay({mafiaKill, doctorSave, usersData, usersCollection,
     const minutes = today.getUTCMinutes();
     room.update({
       dayVote: firebase.firestore.FieldValue.arrayUnion({
+        player: userUid,
```

Comment:
Are you using this? It seems to me that you can use the users collection chose or this, I don't think you need both.